### PR TITLE
Issue #8, Tests topic topic element [part 1]

### PR DESCRIPTION
**Pull request by andrii-z4i**
Covered next methods:
- __init__
- _get_title
- _get_markerrefs
- _get_position
- _get_children
- _set_hyperlink
- getOwnerSheet

### CODE REVIEW

File: xmind/tests/test_topic_element.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            ('_get_children', 0),
+            ('_set_hyperlink', 1),
+            ('getOwnerSheet', 0),
+            ('getTitle', 0),
+            ('setTitle', 1),
+            ('getMarkers', 0),
+            ('addMarker', 1),
+            ('setFolded', 0),
+            ('getPosition', 0),
+            ('removePosition', 0),
```
Comment (mykola-pysarev) [inline xmind/tests/test_topic_element.py]: set_position is missed

File: xmind/tests/test_topic_element.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+    def test_init_has_node_has_no_owner_workbook(self):
+        _element = TopicElement(3)
+        self._workbook_mixin_element_init.assert_called_once_with(3, None)
+        self._add_attribute.assert_called_once_with(ATTR_ID)
+
+    def test_init_has_node_has_owner_workbook(self):
+        _element = TopicElement(3, 5)
+        self._workbook_mixin_element_init.assert_called_once_with(3, 5)
+        self._add_attribute.assert_called_once_with(ATTR_ID)
```
Comment (mykola-pysarev) [inline xmind/tests/test_topic_element.py:99]: shoul we also call it with 3 parameters?

File: xmind/tests/test_topic_element.py (hunk 1 above)
Comment (andrii-z4i) [reply xmind/tests/test_topic_element.py]: added

File: xmind/tests/test_topic_element.py (hunk 2 above)
Comment (andrii-z4i) [reply xmind/tests/test_topic_element.py:99]: Addressed

File: xmind/tests/test_topic_element.py [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        _get_owner_workbook_mock = patch.object(
+            _element, 'getOwnerWorkbook').start()
+
+        _get_parent_node_mock.return_value = None
+
+        self.assertIsNone(_element.getOwnerSheet())
+
+        _get_parent_node_mock.assert_called_once_with()
+        _get_owner_workbook_mock.assert_not_called()
+        self._assert_init_methods()
```
Comment (mykola-pysarev) [inline xmind/tests/test_topic_element.py:153]: should we also add check that parent.parentNode was not call

**Review by mykola-pysarev — APPROVED** [review]

File: xmind/tests/test_topic_element.py (hunk 3 above)
Comment (andrii-z4i) [reply xmind/tests/test_topic_element.py:153]: We don't provide object on which we will test it. If object (parent) is None then None.parentNode can't be called.

**Pull request merged**